Train-In-Blocks-Ltd/Landing
# feat: Add testimonials to home and compare pages

# TIB Landing PR

## Changes

A brief list of changes

- Made a new component ```Testimonials```
- Add this new comp to home and compare pages

## Test Cases

What should the reviewer do and expect to happen when testing

1. See testimonial on home and compare pages. **Expect to see 2 column grid with text**
2. Resize the window to check responsiveness. **Expect to see single column on smaller size**

## Demo

<img width="1660" alt="Screenshot 2022-03-27 at 11 50 32" src="https://user-images.githubusercontent.com/63963445/160278236-edb2f1cf-f6ad-4946-8276-012d041a3770.png">
<img width="676" alt="Screenshot 2022-03-27 at 11 56 13" src="https://user-images.githubusercontent.com/63963445/160278240-ed6cebe1-5df8-4c8c-bd79-9b348d842ba7.png">
